agent/functional_tests/testdata/taskdefinitions/awslogs-datetime-windows/task-definition.json

```diff
@@ -0,0 +1,21 @@
+{
+  "family": "ecs-awslogs-datetime-windows-test",
+  "containerDefinitions": [{
+    "essential": true,
+    "memory": 10,
```

Comment:
I'd suggest you increase the memory to >256, as low memory could cause the test failure due to slow start/stop, related to this [issue](https://github.com/aws/amazon-ecs-agent/issues/1096). Same for the other task definition.